src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -1596,6 +1597,16 @@ public Territory getCurrentBattleTerritory() {
   public IBattle getCurrentBattle() {
     return m_currentBattle;
   }
+
+  protected void raids() {
+    // Remove air raids - this adds all the bombing raids to the battle tracker
+    for( final Territory t : m_battleTracker.getPendingSBRSites() ) {
```

Comment:
It follows logically from a user viewpoint here. All the other defender decisions are here. Meaning kamikaze and scramble, do you really want intercept somewhere else? 

I can mark the method private. I don't see positive value from changing the location from what I have proposed and tested.